docker-compose.yml

```diff
@@ -51,8 +52,6 @@ services:
   #     nofile:
   #       soft: 65536
   #       hard: 65536
-  #   ports:
-  #     - '127.0.0.1:9200:9200'
```

Comment:
Doesn't this expose the ES to the internet?

`external_network` is kinda required for the geoip downloader, but if you're disabling that... you can remove the external part.